bin/create-release

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env python3
+
+import os
+import re
+import requests
+import subprocess
+import shlex
+import sys
+
+# DOC: Create a new draft CiviForm image release. Usage: bin/create-release COMMIT_SHA RELEASE_NUMBER
+#
+# Requirements:
+#   Must have creds to login to docker hub as "civiform" user.
+#   Must have a GitHub API token with repo and read:org scopes located at "gh-release-token.txt"
+#
+# When successful, this script:
+#   1. tags the git commit for the release with the release number
+#   2. tags the docker image for the release with the release number
+#   3. creates a new draft release on github
+
+if not os.path.exists("gh-release-token.txt"):
+    print("gh-release-token.txt not found in root directory", file=sys.stderr)
+    print(
+        "Create one with repo and read:org scopes at https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token",
+        file=sys.stderr)
+    exit(1)
+
+if len(sys.argv) != 3:
+    print(
+        "Usage: bin/create-release COMMIT_SHA RELEASE_NUMBER", file=sys.stderr)
+    exit(1)
+
+
+def shell_cmd(command_string):
+    try:
+        process = subprocess.run(
+            shlex.split(command_string), capture_output=True)
+    except FileNotFoundError:
+        return ''
+    return process.stdout.decode('ascii') if process else ''
+
+
+################################################################################
+# Resolve and validate GitHub token, check auth
+################################################################################
+
+with open("gh-release-token.txt", "r") as file:
+    GH_TOKEN = file.read().rstrip()
```

Comment:
Not blocking: Any particular reason to use UPPER_CAMEL_CASE here rather than lower_camel_case? If I recall correctly, the former is typically used to represent package-level constants rather than things that are dynamically initialized.

Reply:
i was using constant case for every variable that doesn't change after initialization. would you prefer i downcase this and the other vars (e.g. the release number, sha, etc)?

Reply:
My preference is weak and your rationale makes sense. I'm fine keeping it this way, it was just a new pattern to me.

Reply:
We should follow the style guide for this: https://google.github.io/styleguide/pyguide.html#316-naming

Global/class level constants get CAPS_WITH_UNDER, as do "module-level constants" (https://google.github.io/styleguide/pyguide.html#254-decision)

Reply:
I don't believe this counts, in the sense that it's not a fixed value for the module

Reply:
downcased it